.github/workflows/internal.yml

```diff
@@ -0,0 +1,33 @@
+name: Internal-unit-tests
```

Comment:
❓ does the `name` have to match the yml file name or does that not matter?

Reply:
technically it doesn't, but for consistency makes sense to me to keep it the same, changed here:
https://github.com/stellar/stellar-etl/pull/130/commits/47bbd8c40ad6c0c60720b4d826c25d571da4a94a